src/main/java/uk/gov/hmcts/reform/blobrouter/services/EnvelopeService.java

```diff
@@ -92,7 +93,12 @@ public void markAsRejected(UUID id, String reason) {
                 env -> {
                     envelopeRepository.updateStatus(id, Status.REJECTED);
                     envelopeRepository.updatePendingNotification(id, true); // notification pending
-                    eventRepository.insert(new NewEnvelopeEvent(id, EventType.REJECTED, reason));
+                    eventRepository.insert(new NewEnvelopeEvent(
+                        id,
+                        EventType.REJECTED,
+                        ErrorCode.ERR_AV_FAILED,
```

Comment:
why this code?

Reply:
The error code is needed for the notification message to let Exela know why it is failed. Notification Service which consumes the messages from notification is expecting the `error_code` with ErrorCode (enum)

Reply:
I mean, why `ERR_AV_FAILED`?

Reply:
it can be any error type, doesn't have to be `ERR_AV_FAILED`

Reply:
oh sorry, I didn't notice that. thought it's a test. will update

Reply:
This is causing several changes. For now, I've set the value as null. 
In the next PR, I'll add error_code parameter to the method, and update the references.